Wiki/zh-Tw.js

```diff
@@ -0,0 +1,678 @@
+  WIKI_KEYS: {
+    TITLE: "指南小提示",
+    DESCRIPTION: "在每個網頁會有一個小提示，它們可以讓您考慮一些重要的資訊。 您可以在下面範例找到這裡提到的一些小提示",
+    SUCCESS_KEY: "說明或建議事項將會顯示在與此類小提示類似的框中",
+    INFO_KEY: "信息或註解將會顯示在與此類小提示類似的框中",
+    WARNING_KEY: "警告將會顯示在與此類小提示類似的框中",
+    DANGER_KEY: "重要訊息將會顯示在與此類小提示類似的框中"
+  },
+  REQUEST_TRANSLATION: {
+    TITLE: "協助翻譯或修正內容！",
+    DESCRIPTION: "如果您需要，您可以要求 更新 或 修改 這份指南的任何主題。無論是為了改進還是為了更正過時或已棄用的用詞" + "\n" + "這份指南擁有著許多語言，都是志願者提供的幫助。如果您想要協助我們翻譯，**[請點擊這裡]({{link}})**"
+  }
+};
+
+exports.HELPFUL_LINKS = {
+  TITLE: "實用連結"
+};
+
+exports.NAVBAR = {
+  HOME: "指南首頁",
+  DISCORD_SERVER: "Discord 伺服器",
+  PRIVACY_POLICY: "隱私政策",
+  TERMS_AND_CONDITIONS: "條款及細則",
+  WEBSITE: "官方網站",
+  DONATE: "捐款",
+  BLOG: "部落格",
+  TWITTER: "Twitter",
+  VOTE: "投票",
+  GETTING_STARTED: "入門",
+  INVITE_FILO: "邀請 Filo",
+  MODULES: "模組",
+  LANGUAGE: "語言",
+  PREFIX: "Prefix",
+  GUILD: "伺服器",
+  REPORTS: "檢舉",
+  WELCOMES: "歡迎",
+  WELCOME_ROLE: "預設身分組",
+  FAREWELLS: "離別訊息",
+  MEMBER_COUNTER: "人數統計",
+  MEMBER_COUNTER_NICKNAME: "人數統計 (暱稱)",
+  LOGGING: "日誌",
+  ANTI_INVITES: "反邀請",
+  AUTO_MODERATION: "AUTO MODERATION",
+  ANTI_EVASION: "反規避",
+  ANTI_SPAM: "反洗版",
+  ANTI_MASS_MENTIONS: "防止提及",
+  ANTI_REPEATED_TEXT: "返重複文字",
+  DISABLE_COMMANDS: "停用指令",
+  ANTI_BAD_WORDS: "反壞話",
+  COMMANDS: "指令",
+  TOPICS: "主題",
+  CLOUDFLARE: "Cloudflare",
+  COMMON_ERRORS: "常見錯誤"
+};
+
+exports.HOME = {
+  NAVIGATION_INDEX: {
+    TITLE: "導覽目錄",
+    GETTING_STARTED: {
+      DESCRIPTION: "如果你剛加入 Filo 的行列，你可以從下面的連結查看必要資訊"
+    }
+  },
+  MODULES: {
+    DESCRIPTION: "在這裡您可以找到各種關於模組的重要資訊"
+  }
+};
+
+exports.GETTING_STARTED = {
+  INVITE_FILO: {
+    DESCRIPTION: "接下來你會看到一個伺服器邀請的例子，還有我們建議的權限設定",
+    PERMISSIONS_NEEDED: "你必須是伺服器 **擁有者** 或是擁有這些權限： {{permission_1}} 或 {{permission_2}} 才能將 Filo 邀請至伺服器",
+    SLASH_COMMAND_WARNING: "目前 Filo 將不支援沒有把機器人斜線指令權限開啟的伺服器，或是註冊 Filo 斜線指令但把 Filo 踢出的伺服器，如此一來你將會收到一個一般錯誤",
+    STEP_1: {
+      TITLE: "打開瀏覽器中的邀請連結",
+      DESCRIPTION: "**[點擊這裡]({{link}})** 我們將會為您導向到邀請頁面和斜線指令權限頁",
+      EXAMPLE: "邀請頁面",
+      ALTERNATIVE_INVITE: "如果你不需要斜線指令的話請點擊 **[這裡的邀請連結]({{link}})**"
+    },
+    STEP_2: {
+      TITLE: "選擇 Filo 要落腳的伺服器",
+      DESCRIPTION: "打開 **新增至伺服器** 的下拉式選單並選擇要添加 Filo 的伺服器",
+      EXAMPLE: "添加伺服器的選擇框 (標註部分)",
+      BEFORE_THE_RESULT_EXAMPLE: "當你選擇好之後，你應該會看到下圖的畫面",
+      CONTINUE: "當完成這個步驟後，你應該會看到 **繼續** 按鈕。不用猶豫了，點下去吧！"
+    },
+    STEP_3: {
+      TITLE: "選擇權限 (可選可不選)",
+      DESCRIPTION: "這個步驟可以讓您選擇 Filo 的伺服器權限 (非頻道權限). 如果沒有特別要求，請點擊 **繼續** 按鈕",
+      INTEGRATION_ROLE: "如果您全選了所有權限，Discord 會幫您建立一個 **Filo**的身分組。反之，Discord 會把權限單獨寫在他的身分裡",
+      PERMISSIONS_NEEDED: "Filo 必須有這些指令才能正常工作: {{permission_1}}、 {{permission_2}}、{{permission_3}} 和 {{permission_4}}"
+    },
+    STEP_4: {
+      TITLE: "授權邀請",
+      DESCRIPTION: "當你選好了權限，點擊 **授權** 按鈕來把 Filo 邀進伺服器"
+    }
+  }
+};
+
+exports.MODULES = {
+  ABOUT_THE_MODULE: "關於本模組",
+  COMMUNITY_RECOMMENDATION: "我們強烈建議你使用本模組來管理社群",
+  TO_ACTION_COMMAND: "要 {{action}} 本模組，請輸入本指令： {{command}}",
```

Comment:
??

Translation does not make sense in all contexts.

The model translation is: "To {{action}} you must execute the following command: {{command}}"